db/migrate/20211118004240_migrate_call_to_actions_to_api.rb

```diff
@@ -0,0 +1,134 @@
+class MigrateCallToActionsToApi < ActiveRecord::Migration[6.1]
+  def change
+    files = Dir[Rails.root + 'app/models/*.rb']
+    models = files.map{ |m| File.basename(m, '.rb').camelize }
+
+    if models.include?("CallToAction")
+      CallToAction.where(cta_type: 'contact-us').each do |cta|
+      p 'Creating api namespace'
+
+      params = {
+          name: 'inquiries',
+          requires_authentication: false,
+          version: 1,
+          properties: {
+              "name": cta.name_placeholder,
+              "email": cta.email_placeholder,
+              "phone_number": cta.phone_placeholder,
+              "message": cta.message_placeholder
+          }.to_json
+      }
+
+      p 'Creating api form'
+      api_namespace = ApiNamespace.create(params)
+      api_form_params =  { 
+          properties: { 
+              "name": {
+                  "label": cta.name_label,
+                  "pattern": "",
+                  "required": "0",
+                  "field_type": "input",
+                  "prepopulate": "0",
+                  "type_validation": "free text"
```

Comment:
@Pralish this migration script should be updated. Since its only been run on staging we should be fine to update this one in-place.